test/_java.sh

```diff
@@ -24,8 +24,8 @@ printf "\n"
 java -version
 printf "\n"
 
-echo "update-alternatives --set java /usr/lib/jvm/java-9-oracle/bin/java"
-update-alternatives --set java  /usr/lib/jvm/java-9-oracle/bin/java 
+echo "update-alternatives --set java /usr/lib/jvm/java-10-oracle/bin/java"
+update-alternatives --set java  /usr/lib/jvm/java-10-oracle/bin/java 
```

Comment:
lets change thsi to use `shipctl` to switch java versions. since `shipctl` is already installed on the base image,  we should be using that instead of switching manually